pkg/platform/allocator/vacate_test.go

```diff
@@ -599,6 +625,27 @@ func TestCheckVacateFailures(t *testing.T) {
 				errors.New("cluster [123456789][apm] failed vacating, reason: code: unknown, message: an apm error message"),
 			}},
 		},
+		{
+			name: "Returns na apm error on Kibana vacate failure",
+			args: args{
+				failures: &models.MoveClustersDetails{
+					AppsearchClusters: []*models.MoveAppSearchDetails{
+						{
+							ClusterID: ec.String("123456789"),
```

Comment:
How this doesn't fail with cluster ID validation. I haven't dived into the test a lot. It just alarmed my brain. Ignore if I'm just drunk

Reply:
This test is for CheckVacateFailures, it's not verifying any Cluster IDs